src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,20 @@
 // Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    email,
+    name,
+  } = user;
```

Comment:
```suggestion
  const { email, name } = user;
```
You can leave it in one line if there < 3 props and line is not too long